### DISCUSSION TITLE
Disabled scroll also disables zoom.

### DISCUSSION THREAD

**Discussion by thedipsy** [patrykandpatrick/vico#296] [Ideas]
**Describe the bug**
The horizontal scroll is disabled and also the zoom is disabled too, which I do not want to happen. I would like to keep the zoom feature.

**Module**

- [ x] `compose`
- [ ] `views`
- [ ] `compose-m2`
- [ ] `compose-m3`
- [ ] `core` (select if none of the above apply)

**To reproduce**
Steps to reproduce the behavior.
```
ProvideChartStyle {
    Chart(
      modifier = Modifier.fillMaxWidth(),
      model = data,
      chart = lineChart(
        lines = listOf(
          LineChart.LineSpec(
            lineColor = petrol6.toArgb(),
            lineThicknessDp = 1.75f,
            lineBackgroundShader = DynamicShaders.fromBrush(
              brush = Brush.verticalGradient(
                colorStops = arrayOf(
                  0.0f to  petrol6,
                  0.67f to petrol1
                )
              )
            )
          )
        )
      ),
      chartScrollSpec = rememberChartScrollSpec(isScrollEnabled = false)
    )
  }
```

**Expected behavior**
Horizontal scroll should be disabled, and zoom should be kept.

**Screenshots**
![Screenshot 2023-06-26 at 13 40 47](https://github.com/patrykandpatrick/vico/assets/78094045/92cd7140-2e41-4705-a01c-f18c6854f191)

**Android versions**
Tested on Android 13.

**Comment by patrickmichalik** [comment]
Hello! I’m afraid this isn’t possible in the moment, but we’ll be looking into adding such functionality.

**Comment by patrickmichalik** [comment]
[Vico 2.0.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.9) makes this possible. To achieve the desired result, set `initialZoom` (in `rememberVicoZoomState`) to `Zoom.Content`, keeping scroll enabled. The chart will be scrollable when zoomed in, which, as I understand, it’s supposed to be. Otherwise, you’d have to zoom out and zoom back in to see a different portion of the chart, and it would be hard to obtain an enlarged view of the start or the end of the chart, as you’d have to pinch at the very edge. If scroll should, in fact, be disabled at all times, please let me know. Cheers!

**Comment by Gowsky** [comment]
Hello. [Vico 2.0.0](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0) is now available.
